src/screens/setup/create/ConfirmPassphrase.js

```diff
@@ -44,6 +44,8 @@ class ConfirmPassphrase extends Component {
         }
         shuffle(orderArray);
         for (let i = 0; i < words.length; i++) {
+            //console.log(orderArray[i]);
+            // state.scrambledTags.push({"word":orderArray[i], "active":false});
             state.scrambledTags.push({"wordItem":orderArray[i], "selected": false});
         }
         this.setState(state)
```

Comment:
why does this function grab state? componentDidMountalready only executes on mounting, thus the state would be empty. i think you can remove line 39 and make 51 `this.setState({ orderArray })`